src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingSolutionCrawlerRegistrationService.cs

```diff
@@ -70,7 +70,9 @@ public IUnitTestingWorkCoordinator Register(Solution solution)
                     coordinator = new UnitTestingWorkCoordinator(
                         _listener,
                         GetAnalyzerProviders(workspaceKind),
+#if false // Not used in unit testing crawling
                         initializeLazily: true,
+#endif
```

Comment:
as this is always passed 'true' in, it's not needed as an actual customization flag and has been removed.